src/platform/packages/shared/kbn-alerts-ui-shared/src/add_message_variables/index.tsx

```diff
@@ -58,12 +58,17 @@ export const AddMessageVariables: React.FunctionComponent<Props> = ({
     [messageVariables]
   );
 
+  const hasDeprecatedVariables = useMemo(
```

Comment:
Added this as per @joana-cps's suggestion to automatically hide the deprecated vars footer in case there are none